packages/web/src/components/MapView.tsx

```diff
@@ -11,6 +11,12 @@ const defaultMapOptions = {
     },
     zoom: 13,
 };
+const mapOptions = {
```

Comment:
I think it would be better if we put the mapOptions in `MapViewState` and avoid using global variables.  Did you try that?  Since you'd need to use `this.setState()`, you could just move the entire `getMapBounds()` as a member function of the `MapView` class.

Reply:
No, I haven't thought about that. I'll try that.

Reply:
If I add mapOptions to MapViewState, I get errors in getDerivedStatefromProps where when I create a polyline, it wants to also set the center property.

Reply:
When returning?  Is mapOptions set to be nullable?  Something like 

```javascript
interface MapViewState {
    map?: google.maps.Map;
    maps?: typeof google.maps;
    polyline?: google.maps.Polyline;
    lat?: number,
    lng?: number,
}
```